sky/shell/platform/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -0,0 +1,57 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "sky/shell/platform/ios/framework/Headers/FlutterAppDelegate.h"
+#include "sky/shell/platform/ios/framework/Headers/FlutterViewController.h"
+
+@implementation FlutterAppDelegate {
+  UIWindow* _window;
```

Comment:
It was in the header because the `window` property was added to the protocol in iOS 5.0. Before then, it was a matter of convention that that the app delegate knew about the (usually key) window. Since we don't support < iOS 5.0 anyway, you can get rid of the header definition and just `@synthesize`. Either way.